src/patches.rs

```diff
@@ -13702,6 +13706,42 @@ fn build_and_run_patches(gc_disc: &mut structs::GcDisc, config: &PatchConfig, ve
     Ok(())
 }
 
+fn patch_required_artifact_count(patcher: &mut PrimePatcher, artifact_count: u32) {
+    if artifact_count > 12 {
+        panic!("Must specify between 0 and 12 required artifacts");
+    }
+
+    patcher.add_scly_patch(
+        resource_info!("07_stonehenge.MREA").into(),
+        move |_patcher, area| {
+            let scly = area.mrea().scly_section_mut();
+
+            let layer = &mut scly.layers.as_mut_vec()[1]; // Monoliths and Ridley
```

Comment:
Ulti has a function for getting layers by name now, we should probably use it

Reply:
Done